cli/module_loader.rs

```diff
@@ -78,12 +78,8 @@ impl CliModuleLoader {
     specifier: &ModuleSpecifier,
     maybe_referrer: Option<ModuleSpecifier>,
   ) -> Result<ModuleCodeSource, AnyError> {
-    if specifier.as_str() == "node:module" {
-      return Ok(ModuleCodeSource {
-        code: deno_runtime::deno_node::MODULE_ES_SHIM.to_string(),
-        found_url: specifier.to_owned(),
-        media_type: MediaType::JavaScript,
-      });
+    if specifier.scheme() == "node" {
+      unreachable!("Node built-in modules should be handled internally.");
```

Comment:
Oh, the tests are currently panicking on this.

Reply:
Disabled the assertion for now